src/common/ipfixlolib/ipfixlolib.c

```diff
@@ -1316,9 +1316,6 @@ static int valid_transport_protocol(enum ipfix_transport_protocol p) {
 #ifdef IPFIXLOLIB_RAWDIR_SUPPORT 
```

Comment:
The problem is that RAWDIR exists only if that define is set:

```
src/common/ipfixlolib/ipfixlolib.c:1316:7: error: ‘RAWDIR’ undeclared (first use in this function)
  case RAWDIR:
```
src/common/ipfixlolib/ipfixlolib.h:

```
enum ipfix_transport_protocol {
#ifdef IPFIXLOLIB_RAWDIR_SUPPORT 
	RAWDIR, 
#endif 
```